README.md

```diff
@@ -35,22 +35,13 @@ limitations under the License.
 `@elyra/pipeline-editor` is available as an [npm package](https://www.npmjs.com/package/@elyra/pipeline-editor):
 
 ```sh
-// npm
+# npm
 npm install @elyra/pipeline-editor
 
-// yarn
+# yarn
```

Comment:
Do we need to build at some time? Could we add that on the readme as well.

Reply:
Since we don't really have an established build process yet and we have an awkward mix on make and npm commands in the project, I decided to keep the readme as instructions on using the npm published package and usage of the component until we iron some things out

Reply:
If someone like me, which is not completely familiar with the project, comes to the project and wants to build it and use it, where are the instructions? For now, I don't care if it's using make or yarn (well, make would have been better)  but we should have that somewhere...

Reply:
It's a bit tricky to build and use right now... It has to be linked to another project like jupyter/vscode. For vscode you should have to build anything, because vscode builds it, but I think it's a bit different in jupyterlab. I was going to wait until we iron out the process before documenting it